profiler/src/main/java/com/splunk/opentelemetry/profiler/JfrActivator.java

```diff
@@ -38,5 +44,20 @@ public void afterByteBuddyAgent(Config config) {
       return;
     }
     logger.info("JFR profiler is active.");
+    executor.submit(logUncaught(() -> activateJfrAndRunForever(config)));
+  }
+
+  private void activateJfrAndRunForever(Config config) {
+    String recordingDurationStr = config.getProperty(CONFIG_KEY_RECORDING_DURATION_SECONDS, "20");
```

Comment:
any particular reason 20s is a good default?

Reply:
Based on the events we plan on capturing and the frequency of those events, 20s seems like a reasonable time to keep the data volume lowish while also being timely.  This can be squishy I think.